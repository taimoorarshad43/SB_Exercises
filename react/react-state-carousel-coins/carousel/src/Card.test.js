import { render } from "@testing-library/react";
import Card from "./Card";

// Smoke test for Card component
it("renders without crashing", function() {
  render(
    <Card
      caption="Test caption"
      src="test.jpg"
      currNum={1}
      totalNum={3}
    />
  );
});

// Snapshot test for Card component
it("matches snapshot", function() {
  const { container } = render(
    <Card
      caption="Test caption"
      src="test.jpg"
      currNum={1}
      totalNum={3}
    />
  );
  expect(container).toMatchSnapshot();
});
