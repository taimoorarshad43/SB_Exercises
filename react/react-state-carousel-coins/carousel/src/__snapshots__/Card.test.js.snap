// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Test caption
    </h4>
    <img
      alt="Test caption"
      class="Card-image"
      src="test.jpg"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      3
      .
    </small>
  </div>
</div>
`;
